packages/ui/src/components/va-carousel/_variables.scss

```diff
@@ -0,0 +1,6 @@
+:root {
+  --va-carousel-background: var(--va-background);
+  --va-carousel-border-radius: 10px;
```

Comment:
maybe we can use our global css-variable:
`--va-block-border-radius: 0.375rem;`

Reply:
Would be nice to have more variation of border radius css varaible. `--va-block-border-radius--large`, `--va-block-border-radius--medium`, `--va-block-border-radius--smal`